### PR TITLE
Add const reference in opportunities detected by clang-tidy

### PR DESCRIPTION
**Pull request by cyyever**

### CODE REVIEW

File: torch/csrc/utils/python_dispatch.cpp [hunk 1]
```diff
@@ -192,10 +193,10 @@ static torch::_RegisterOrVerify register_or_verify() {
 static py::object ophandle_call_boxed(
     const c10::OperatorHandle& handle,
     py::args args,
-    py::kwargs kwargs) {
+    const py::kwargs& kwargs) {
   auto stack = torch::jit::createStackForSchema(
       handle.schema(),
-      args,
+      std::move(args),
```
Comment (Skylion007) [inline torch/csrc/utils/python_dispatch.cpp:199]: Hmm, only args should be moved by value? Is this intentional? I guess we just iterate through the kwargs dict then?

**Review by Skylion007 — APPROVED** [review]
Looks good once the lint errors are fixed and all checks pass.

File: torch/csrc/utils/python_dispatch.cpp (hunk 1 above)
Comment (cyyever) [reply torch/csrc/utils/python_dispatch.cpp:199]: kwargs is used as const arguments in the function body, no benefits to move

**Pull request closed without merging**